Userland/Libraries/LibJS/Runtime/TypedArrayPrototype.cpp

```diff
@@ -162,6 +168,69 @@ JS_DEFINE_NATIVE_FUNCTION(TypedArrayPrototype::every)
     return Value(result);
 }
 
+// 23.2.3.8 %TypedArray%.prototype.fill ( value [ , start [ , end ] ] ), https://tc39.es/ecma262/#sec-%typedarray%.prototype.fill
+JS_DEFINE_NATIVE_FUNCTION(TypedArrayPrototype::fill)
+{
+    auto typed_array = typed_array_from(vm, global_object);
+    if (!typed_array)
+        return {};
+
+    auto length = typed_array->array_length();
+
+    Value value;
+    if (typed_array->content_type() == TypedArrayBase::ContentType::BigInt) {
+        value = vm.argument(0).to_bigint(global_object);
+        if (vm.exception())
+            return {};
+    } else {
+        value = vm.argument(0).to_number(global_object);
+        if (vm.exception())
+            return {};
+    }
+
+    auto relative_start = vm.argument(1).to_integer_or_infinity(global_object);
+    if (vm.exception())
+        return {};
+
+    u32 k;
+    if (Value(relative_start).is_negative_infinity())
+        k = 0;
+    else if (relative_start < 0)
+        k = max(length + relative_start, 0);
+    else
+        k = min(relative_start, length);
+
+    double relative_end;
+    if (vm.argument(2).is_undefined()) {
+        relative_end = length;
+    } else {
+        relative_end = vm.argument(2).to_integer_or_infinity(global_object);
+        if (vm.exception())
+            return {};
+    }
+
+    u32 final;
```

Comment:
We might want to avoid using C++ keywords as variable names

Reply:
i mean its completely valid (as they are only keywords in the context of class or method declarations), so i dont think changing it to `final_` will achieve much